app/src/main/java/org/p2p/wallet/settings/ui/network/SettingsNetworkFragment.kt

```diff
@@ -66,17 +67,20 @@ class SettingsNetworkFragment :
         presenter.setNewEnvironment(environment)
     }
 
-    override fun showEnvironment(environment: Environment) {
+    override fun showEnvironment(environment: Environment, isDevnetEnabled: Boolean) {
```

Comment:
I think it's not good idea to have isDevNet booleans

Reply:
maybe we will decide is it dev net or not in some class which take as parameter build flavor